discovery_engine_core/core/src/engine.rs

```diff
@@ -12,7 +12,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use std::{collections::HashMap, sync::Arc};
+use std::{collections::HashMap, sync::Arc, usize};
```

Comment:
i guess this was accidentally added by the IDE? it's in the prelude anyways.
```suggestion
use std::{collections::HashMap, sync::Arc};
```

Reply:
i guess it was added accidentally, good catch.